### ISSUE TITLE
html custom properties with preserve: false

### ISSUE THREAD

**Issue by jzempel** [postcss/postcss-custom-properties#142]
With v8.0.0 support for custom properties declared within `html` rules, we lost the ability to reference variables under `html` with `preserve: false`. Given the following:

```css
:root {
  --html-background-color
}

html {
  background-color: var(--html-background-color);
}
```

After postcss (with `preserve: false`):

```css
html {
  background-color: var(--html-background-color);
}
```

Is there a workaround available? Here's the actual source file we are attempting to process: https://github.com/zendeskgarden/css-components/blob/master/packages/bedrock/src/_base.css. All custom properties under the `html` selector are not being resolved like they were prior to v8.0.0.

**Comment by jonathantneal** [comment]
Great catch, @jzempel. I’m sorry you had to find the bug, but it’s been resolved in 8.0.6. 🎉

**Comment by jzempel** [comment]
We'll keep :eyes: out for when you roll that patch into https://github.com/csstools/postcss-preset-env. Thanks!

**Comment by jonathantneal** [comment]
Of course. It has shipped in 6.0.7.
